net/client.go

```diff
@@ -37,3 +39,8 @@ type PublicClient interface {
 	Group(ctx context.Context, p Peer, in *drand.GroupRequest) (*drand.GroupPacket, error)
 	Home(ctx context.Context, p Peer, in *drand.HomeRequest) (*drand.HomeResponse, error)
 }
+
+// HTTPClient is an optional extension to the protocol client allowing relay of metrics
```

Comment:
Well specifically this is just a generic handler that can handle anything at this package. It's only used for metrics so far.